### ISSUE TITLE
[Accessibility]: Acronym for commit author in icon is pronounced

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35256]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33513

### Problem description

In the list of commits, we display an icon with the acronym of the author as text inside. For "Istio Automation", this becomes "Ia", pronounced "eeh-aah" by the screen reader. This is confusing and at best the icon adds nothing for the screen reader.

![image](https://user-images.githubusercontent.com/18282288/167685381-8230e1ce-51c1-4d70-8f1b-4e3340dacdb6.png)

### Expected behavior

The Icon would probably be skipped entirely, with the screen reader advancing to the author name

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by umpox** [comment]
Will be closed by https://github.com/sourcegraph/sourcegraph/pull/35617
